app/routes/events_.$eventId.tsx

```diff
@@ -126,3 +126,12 @@ export function ErrorBoundary() {
   const { eventId } = useParams();
   return <div className="error-container">There was an error loading event by the id {eventId}.</div>;
 }
+
+export const meta: MetaFunction<typeof loader> = ({ data }) => {
+  const eventFullName = data.event.name;
+  const regex = /^[^:]+/;
```

Comment:
Not a blocker: Why do we need regex? If it's necessary, should we place a tiny comment on what it does?

Reply:
If there's a title in the format 'Tech Talks: Exploring the Latest Web Development Trends', it will only take the text before the colon.